Moving Hazel Hen to archive, structure, and GitHub smileys (#73)
precice/precice.github.io: pages/docs/installation/installation-special-systems.md

```bash
Currently Loaded Modulefiles:
 1) admin/1.0   2) tempdir/1.0   3) lrz/1.0   4) spack/21.1.1   5) gcc/8.4.0   6) intel-mpi/2019-gcc   7) precice/2.2.0-gcc8-impi
```

**Note:** If you want to use FEniCS (see below), please stick to GCC from the very beginning.

##### Building with CMake

> **warning** This page needs updates for preCICE v2 and the module system rolled out on CooLMUC in June 2021

Reviewer: ```suggestion
{% include warning.html content=" This page needs updates for preCICE v2." %}
```

Run this, if you log in and everything has already been prepared as described above:

```bash
module unload intel-mpi/2019-intel intel-mkl/2019 intel/19.0.5
module load gcc/8 intel-mpi/2019-gcc precice/2.2.0-gcc8-impi
source activate pyprecice
```

### Cartesius (Dutch national supercomputer)

#### modules and environment

```bash
module load 2020
module load CMake/3.16.4-GCCcore-9.3.0 PETSc/3.12.4-foss-2020a-Python-3.8.2 Eigen/3.3.9-GCCcore-9.3.0 ScaLAPACK/2.1.0-gompi-2020a
```

After loading these modules you can proceed with the `cmake` build steps for preCICE.

For python bindings,

```bash
CPATH=<PRECICE_DIR>/include/ pip install pyprecice
```

Replace `PRECICE_DIR` with the installation prefix used for preCICE. Also, make sure that preCICE libraries locations are in `LD_LIBRARY_PATH` and `LIBRARY_PATH`.

## Archived systems

### Hazel Hen (Cray/Intel, Stuttgart)

> **warning** This page needs updates for preCICE v2

Reviewer: We have something different for warnings:
```suggestion
{% include warning.html content=" This page needs updates for preCICE v2." %}
```

Use `ipogif0` for socket communication.

### SuperMUC (Lenovo/Intel, Munich)

> **warning** This page needs updates for preCICE v2

Reviewer: ```suggestion
{% include warning.html content=" This page needs updates for preCICE v2." %}
```

### MAC Cluster (various architectures, Munich)

> **warning** This page needs updates for preCICE v2

Reviewer: ```suggestion
{% include warning.html content=" This page needs updates for preCICE v2." %}
```

### MareNostrum (Lenovo/Intel, Barcelona)

> **warning** This page needs updates for preCICE v2

Reviewer: ```suggestion
{% include warning.html content=" This page needs updates for preCICE v2." %}
```
